### PR TITLE
Remove reference to tradfri groups and manual configuration

### PR DESCRIPTION
**Pull request by ggravlingen**
## Proposed change

We are deprecating the use of automated import of groups in `core`. This also means we are deprecating the use of manual configuration. This PR updates the documentation to reflect those changes.

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [x] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: https://github.com/home-assistant/core/pull/65226
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: 

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [ ] The documentation follows the Home Assistant documentation [standards][].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

File: source/_integrations/tradfri.markdown [hunk 1]
```diff
@@ -29,28 +29,6 @@ You will be prompted to configure the gateway through the Home Assistant interfa
 If you see an "Unable to connect" message, restart the gateway and try again. Don't forget to assign a permanent IP address to your IKEA Trådfri Gateway on your router or DHCP server.
 </div>
```
Comment (MartinHjelmare) [inline source/_integrations/tradfri.markdown:33]: Please include our config flow template instead.
```suggestion

{% include integrations/config_flow.md %}

```

**Review by MartinHjelmare — DISMISSED** [review]
Thanks!

**Review by MartinHjelmare — COMMENTED** [review]
Should we move this section `You will be prompted to configure...` including the note under the config flow template? Then it will be included under the configuration header of the template.

**Comment by ggravlingen** [comment]
> Should we move this section `You will be prompted to configure...` including the note under the config flow template? Then it will be included under the configuration header of the template.

That makes sense. Sorry, might have dismissed a review suggestion from you that I didn't notice before I pushed.

**Review by MartinHjelmare — APPROVED** [review]
Great!

**Pull request merged**